utils/rpms/daos.spec

```diff
@@ -362,9 +362,12 @@ getent passwd daos >/dev/null || useradd -M daos
 %{_libdir}/*.a
 
 %changelog
-* Tue Jun 2 2020 Hua Kuang <hua.kuang@intel.com> - 1.1.0-20
+* Thu Jun 4 2020 Hua Kuang <hua.kuang@intel.com> - 1.1.0-21
```

Comment:
this will fail rpmlint validation, date needs to be 2 digits, i.e. "Thu Jun 04 2020"

Reply:
thanks, will fix it now.